paddle/fluid/platform/device_tracer.cc

```diff
@@ -54,6 +55,36 @@ uint64_t kAlignSize = 8;
     }                                                                      \
   } while (0)
 
+std::string MemcpyKind(CUpti_ActivityMemcpyKind kind) {
+  switch (kind) {
+    case CUPTI_ACTIVITY_MEMCPY_KIND_HTOD:
+      return "MEMCPYHtoD";
```

Comment:
Maybe just `HtoD` is ok? There are many duplicates `MEMCPY`

Reply:
Done

Reply:
I Added it back. User can search MEMCPY to find all copy.